pandas/tests/io/excel/test_writers.py

```diff
@@ -250,6 +250,7 @@ class and any subclasses, on account of the `autouse=True`
         set_option(option_name, prev_engine)  # Roll back option change
 
 
+@td.skip_if_no('xlrd')
```

Comment:
The writer tests still assume xlrd to be there. Will fix up in a follow up to address proper parametrization here (have focused more on readers given recent PRs)

@simonjayhawkins